src/schedule.rs

```diff
@@ -1,6 +1,12 @@
 use super::NifEnv;
 use super::wrapper::nif_interface::enif_consume_timeslice;
 
+pub enum NifScheduleFlags {
+    NORMAL = 0,
```

Comment:
I believe the general rust naming convention is to use CamelCase for enum variants. It's different in `erlang_nif_sys` because I belive it mirrors the naming in the C header.